test_launch_ros/test/test_launch_ros/actions/test_load_composable_nodes.py

```diff
@@ -71,7 +71,10 @@ def __init__(self):
     def load_node_callback(self, request, response):
         self.requests.append(request)
         response.success = True
-        response.full_node_name = f'{request.node_namespace}/{request.node_name}'
+        if request.node_namespace == '/':
+            response.full_node_name = f'/{request.node_name}'
+        else:
+            response.full_node_name = f'{request.node_namespace}/{request.node_name}'
```

Comment:
This change is to avoid illegal node names (containing two forward slashes, e.g. `//test_node_name`).